composer.json

```diff
@@ -34,17 +34,18 @@
     "nette/di": "~3.0.0",
     "nettrine/annotations": "^0.6.0",
     "ninjify/nunjuck": "^0.3",
-    "ninjify/qa": "^0.10",
+    "ninjify/qa": "^0.11",
+    "phpstan/phpstan": "^0.12",
     "phpstan/extension-installer": "^1.0",
-    "phpstan/phpstan-deprecation-rules": "^0.11",
-    "phpstan/phpstan-nette": "^0.11",
-    "phpstan/phpstan-shim": "^0.11",
-    "phpstan/phpstan-strict-rules": "^0.11",
+    "phpstan/phpstan-deprecation-rules": "^0.12",
+    "phpstan/phpstan-nette": "^0.12",
+    "phpstan/phpstan-strict-rules": "^0.12",
     "psr/log": "^1.1",
     "symfony/validator": "^4.1.1|^5.0.0"
   },
   "conflict": {
-    "nette/di": "<=3.0.0-RC"
+    "nette/di": "<=3.0.0-RC",
+    "guzzlehttp/psr7": "<1.7.0"
```

Comment:
There is no need for this now because we require contributte/psr7 0.7+ and it requires psr >1.7.0